apps/server/src/modules/tool/external-tool/controller/dto/response/external-tool-medium-metadata.response.ts

```diff
@@ -0,0 +1,30 @@
+import { ApiProperty, ApiPropertyOptional } from '@nestjs/swagger';
+
+export class ExternalToolMediumMetadataResponse {
```

Comment:
The response does not look like it has much to do with external tools. It is a completely different resource and should not be under /tools/external-tools. This is very medium specific and should be moved to the media-source module.